find_missing_species.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+
+import pandas as pd
+
+
+species_id_col_name = 'Species ID'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Finds rows in trees inventory with species id missing from species_attributes.csv.')
+    parser.add_argument('-u', '--trees-inventory-url',
+        default='https://data.smgov.net/resource/w8ue-6cnd.csv?$limit=50000',
+        help='trees inventory url')
+    parser.add_argument('-s', '--species-attributes-csv',
+        default='data/species_attributes.csv',
+        help='file path for species_attributes.csv')
+
+    return parser.parse_args()
+
+
+def filter_new_species_ids(trees_inventory_df, species_df):
+    """
+    This finds full rows (aka all columns) from trees_inventory_df where
```

Comment:
please document how to run this script (from command line?)

Reply:
Done. See readme.